benchmarks/maxtext_xpk_runner.py

```diff
@@ -265,13 +266,16 @@ def build_user_command(
     cluster_config: XpkConfig,
     base_output_directory: str, 
     buffer_size: int,
+    use_pathways: bool = False,
 ):
   config_tuning_params = ''
   for key, value in model.tuning_params.items():
     config_tuning_params += f'{key}={value} '
 
   install_libtpu_cmd = ''
-  if libtpu_type == LibTpuType.NIGHTLY:
+  if use_pathways:
+    pass
```

Comment:
For future - Should we perhaps give users a message that we won't be running Pathways with nightly Libtpu?